home/web/client/components/AccountsChooser/AccountsChooser.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import DropDownMenu from 'material-ui/DropDownMenu';
 import MenuItem from 'material-ui/MenuItem';
 
-import isEqual from 'lodash.isequal';
```

Comment:
So why we don't use this small packages again? This will include whole lodash for our builds.